jac-uk/apply
# #1313 Location preferences

## What's included?
There were a few issues with the way that location preferences were being collected, to resolve this i've changed the interface for 'ranked-choice' type questions to be perceptive of one choice per rank. 
This streamlines the way things work and negates the issues.
I've also added numbers to review page to show the rank given to each choice

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Apply to an exercise and fill in your location preferences.
- Observe new interface
- find any bugs?
- Check numbers display next to correct location preference on review page

## Risk - how likely is this to impact other areas?
🟠 Medium risk - I've done a sweep and tried to ensure the changes to ranked-choice will remain compatible with other places its used, from what i found there was only one other item using the ranked choice component. The issues with location preferences would have been present here too so i believe it should be okay.

## Additional context
Include screen grabs, notes etc.

![image](https://user-images.githubusercontent.com/44227249/117798326-f0aa1580-b248-11eb-8567-558049a8282d.png)

![image](https://user-images.githubusercontent.com/44227249/117798735-53031600-b249-11eb-989c-4541cd086897.png)

---
PREVIEW:DEVELOP
